v3/src/models/data/attribute.ts

```diff
@@ -117,12 +118,14 @@ export const Attribute = types.model("Attribute", {
   // should be called before retrieving snapshot (i.e. before serialization)
   prepareSnapshot() {
     if (isDevelopment()) {
```

Comment:
For my information: Why is this a no-op in production?

Reply:
See the comment in line 111 above, but the short answer is that MST freezes properties at the JS level in development but not in production (since there's a performance cost), so the implementation is slightly different in the two cases.